scene/2d/physics_body_2d.cpp

```diff
@@ -1103,6 +1158,97 @@ void RigidBody2D::_reload_physics_characteristics() {
 
 //////////////////////////
 
+int RigidCollision2D::get_shape_pair_count() const {
+	return local_shapes.size();
+}
+
+Vector2 RigidCollision2D::get_local_position() const {
+	return local_pos;
+}
+
+Vector2 RigidCollision2D::get_local_normal() const {
+	return local_normal;
+}
+
+Object *RigidCollision2D::get_local_shape(int p_pair_idx) const {
+	ERR_FAIL_COND_V(p_pair_idx >= local_shapes.size(), nullptr);
```

Comment:
```suggestion
	ERR_FAIL_INDEX_V(p_pair_idx, local_shapes.size(), nullptr);
```
The argument is unsigned, so needs to check for negative values, this macro does this and gives a clearer message, same for all these